src/WebServer.Logging/WebServer.Logging.csproj

```diff
@@ -0,0 +1,134 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The main reason of having a separate project + separate dll for this is so that people can publish NuGet packages that depend on Restup.Logging 1.0.0.0 and use them with any Restup that depends on the same logging as well.
We would only have another NuGet package if someone could build on top of it and publish something themselves.